### PR TITLE
[lokijs] Include disableFreeze

### PR DESCRIPTION
**Pull request by sgvictorino**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/techfort/LokiJS/blob/971473a9a9c50f17fe49f3bf26a1873181dbfc9d/src/lokijs.js#L4843
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/lokijs/index.d.ts [hunk 1]
```diff
@@ -1292,6 +1292,7 @@ interface CollectionOptions<E> {
     disableMeta: boolean;
     disableChangesApi: boolean;
     disableDeltaChangesApi: boolean;
+    disableFreeze: boolean;
```
Comment (TeamworkGuy2) [inline types/lokijs/index.d.ts:1295]: This property appears to have been added to Lokijs as part of v1.5.12 (see https://github.com/techfort/LokiJS/commit/971473a9a9c50f17fe49f3bf26a1873181dbfc9d). The version number on line 1 of this definition file should be updated to reflect that the definitions are for a newer version of Lokijs.

Comment (sgvictorino) [reply types/lokijs/index.d.ts:1295]: Oh, thanks!
Should I also update the comment on line 8, or is this reserved for feature updates?
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/ce878d43de147cd006746747d3a24aba58f9c376/types/lokijs/index.d.ts#L8

Comment (TeamworkGuy2) [reply types/lokijs/index.d.ts:1295]: Yes, that would be good to update as well!

**Review by TeamworkGuy2 — APPROVED** [review]

**Review by TeamworkGuy2 — APPROVED** [review]

**Review by andrewbranch — APPROVED** [review]

**Comment by sgvictorino** [comment]
Ready to merge

**Pull request merged**